apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -16,12 +16,14 @@ type Properties<T extends FieldValues> = {
 	errors: FieldErrors<T>;
 	isDisabled?: boolean;
 	isLabelHidden?: boolean;
+	isReadOnly?: boolean;
 	label: string;
 	leftIcon?: JSX.Element;
 	name: FieldPath<T>;
 	placeholder?: string;
 	rightIcon?: JSX.Element;
 	rowsCount?: number;
+	textOverflow?: "clip" | "ellipsis";
```

Comment:
let's remove this, I believe we don't want to use ellipsis for inputs, because we want to be able to copy the value inside it (even without copy button).
```suggestion
```

Reply:
If I'm not mistaken ellipsis also allow users to copy text, but yeah, let's do this to avoid confusion